### PR TITLE
Added Wishlist and cart pages

### PR DESCRIPTION
**Pull request by rishichourasia**
Will work on responsive after the merge.

### CODE REVIEW

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good.

File: components/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+						<i class="far fa-shopping-cart fa-lg"></i>
+					</div>
+				</a>
+			</div>
+			<div class="nav-btn">
+				<a href="/components/login.html"
+					><button class="btn btn-primary">Login</button></a
+				>
+				<a href="/components/signup.html"
+					><button class="btn btn-primary-outlined">Signup</button></a
```
Comment (kr-sarvesh) [inline components/wishlist.html:52]: proper code formating can be done else everything is fine

**Review by kr-sarvesh — COMMENTED** [review]
I liked your user interface part

File: components/cart.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+			<div class="menu-logo">
+				<div class="menu-btn">
+					<a href="/"
+						><img
+							class="logo-svg"
+							src="/assets/logo/logo simple dark.svg"
+							alt="logo-svg"
+					/></a>
+				</div>
+			</div>
```
Comment (dusariudaykumar) [inline components/cart.html:34]: Suggestion Use prettier

**Review by dusariudaykumar — APPROVED** [review]
The UI looks good just one suggestion use code prettier or any plugin for code formatter

**Pull request merged**